### PR TITLE
Remove "also" from SKE content when no scholarships or bursaries

### PR DESCRIPTION
**Pull request by tvararu**

### CODE REVIEW

File: src/Views/Filter/Subject.cshtml [hunk 1]
```diff
@@ -76,7 +76,7 @@
                           @if (subject.IsSubjectKnowledgeEnhancementAvailable)
                             {
                               <p class="govuk-body">
-                                You can also take a <a href="https://getintoteaching.education.gov.uk/explore-my-options/teacher-training-routes/subject-knowledge-enhancement-ske-courses">subject knowledge enhancement (SKE) course</a>.
+                                You can@(subject.Funding?.Scholarship != null || subject.Funding?.BursaryFirst != null ? " also " : " ")take a <a href="https://getintoteaching.education.gov.uk/explore-my-options/teacher-training-routes/subject-knowledge-enhancement-ske-courses">subject knowledge enhancement (SKE) course</a>.
```
Comment (defong) [inline src/Views/Filter/Subject.cshtml]: is it a string can you use string.iswhitespace

Comment (tvararu) [reply src/Views/Filter/Subject.cshtml]: Actually it looks like an `int`

Comment (defong) [reply src/Views/Filter/Subject.cshtml]: HasValue

```csharp
subject.Funding?.Scholarship != null || subject.Funding?.BursaryFirst != null

subject.Funding?.Scholarship.HasValue || subject.Funding?.BursaryFirst.HasValue
```

Comment (tvararu) [reply src/Views/Filter/Subject.cshtml]: Throws `search-and-compare-ui/src/Views/Filter/Subject.cshtml(79,42): error CS0019: Operator '||' cannot be applied to operands of type 'bool?' and 'bool?' :/

**Pull request merged**